package.json

```
@@ -22,7 +22,7 @@
     "test-headless": "karma start --single-run",
     "ci": "npm test && npm run test-headless && npm run coverage && (<coverage/lcov.info coveralls || true) && npm run lint && npm run generate-site",
     "test-browser": "serve .",
-    "coverage": "NODE_ENV=development istanbul cover _mocha -- --reporter dot && echo google-chrome coverage/lcov-report/index.html",
+    "coverage": "nyc npm test",
```

Could we turn on the `text` reporter too and keep echoing the command to view the coverage report?

```suggestion
    "coverage": "NODE_ENV=development nyc --reporter=lcov --reporter=text --all -- npm test && echo google-chrome coverage/lcov-report/index.html",
```

As https://github.com/unexpectedjs/unexpected-stream/blob/39b6dc796c4fbff786fa1cafaccfd792e289c798/package.json#L25

I'm not sure if the `NODE_ENV` thing is necessary, though, but we seem to be passing that everywhere else.

---

@papandreou have added the package.json chunk. Re `NODE_ENV`, in all the places I've implemented nyc it doesn't seem to have any effect, including U-dom, so I'm tempted to keep it off - but if you're still keen I'll add it back :)

---

Okay, cool, let's try without it, then. I'd still like the `&& echo google-chrome coverage/lcov-report/index.html` back, though :)